index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <header>
+    <nav id="navbar">
+      <a href="#welcome-section"> Home </a>
+      <a href="#projects"> Projects </a>
+    </nav>
+  </header>
+  <section id="welcome-section" class="container">
+    <h1> Welcome to our portafolio </h1>
+  </section>
+
+  <section id="projects" class="container">
+    <h1>Our Projects</h1>
+    <div id="all-tiles" class="container">
+      <div class="project-tile">
+        <a href="https://qwibbler.github.io/DanteZaman/" target="_blank" text-decoration: none;>
+          <img class="tile-img" src="img/tp.png"
+            alt="Tribute Page">
+          <h2>Tribute Page</h2>
```

Comment:
- [ ] Remember the headings hierarchy whenever you structure your HTML as this provides better accessibility in the way a browser understands your content. Even if they are in different sections, if possible, try to divide your content importance and thus make use of the headings